### ISSUE TITLE
Explosive can be hidden under the ground model

### ISSUE THREAD

**Issue by Tachii** [KoffeinFlummi/AGM#1961]
Its doesn,t seem right when you are able to hide any explosive under the ground, in a second. I think explosives must be planted above the ground lvl, not under the ground. Or at least you can add an option to burry mines after they are pplanted, and it must take a lot of time.

**Comment by Tachii** [comment]
Also you can glitch explosives through the walls or roofs

**Comment by commy2** [comment]
I don't think we can do anything about that. Arma.

**Comment by BlenderRUS** [comment]
Actually you can check visual line intersection with another objects between eyes and explosive.

**Comment by Tachii** [comment]
Can you use this to do it? https://community.bistudio.com/wiki/lineIntersects

**Comment by commy2** [comment]
>Actually you can check visual line intersection with another objects between eyes and explosive.
>Can you use this to do it?

No. You guys seem to have no clue what actually happens. I don't mean that as insult.
Maybe we can change something about this in future. Marking as feature request.
